utils/block-nameify.ts

```diff
@@ -1,6 +1,7 @@
 import preFormattedBlockNames from './preformatted-block-names.json';
+import preFormattedWords from './preformatted-words.json';
 
-const noFormatting = ['and', 'for', 'of', 'the', 'up', 'with', 'to'];
+const noFormatting = ['and', 'for', 'of', 'the', 'up', 'with', 'to', 'by'];
```

Comment:
```suggestion
const noFormatting = ['and', 'for', 'of', 'the', 'up', 'with', 'to', 'by', 'a'];
```
I found in https://github.com/freeCodeCamp/freeCodeCamp/pull/49813/, that `'a'` was being uppercased, but it's more in this PR's scope than mine!